docs/modules/ROOT/pages/guide-keep3r.adoc

```diff
@@ -47,37 +49,54 @@ image::guide-keep3r-relayer-balance.png[Checking the balance of your Relayer]
+
+If you want to bond KP3R tokens, now is the time to choose how many you want to bond. Note that you can work with zero bonded KP3R tokens, and you can bond more tokens later if you want to do so.
 
-As for the code, paste https://github.com/OpenZeppelin/defender-autotask-examples/blob/master/keep3rs/src/autoregister-keeper.js[this code snippet] from the `defender-autotask-examples` repository. Once you hit save, Defender will start executing the Autotask every minute, taking your Relayer through the bonding and activation process, and then executing available jobs.
+image::guide-keep3r-relayer-bond.png[Bonding your Relayer]
 
-## Wait for Keeper activation
+After the transaction to bond your Relayer is sent, your Relayer will now enter *a 3-day waiting period* before it can be activated as a Relayer. You can access your Relayer page at any time to check how much time is remaining.
 
-The script above will take care of *bonding* your Relayer to the keep3r network using zero KPR tokens as collateral. This is the equivalent of choosing to `Bond` in the https://keep3r.network/[keep3r.network] interface, only that the script is doing it automatically.
+image::guide-keep3r-pending-activation.png[Waiting period for activating the Relayer as a Keeper]
 
-NOTE: Most jobs in the network do not require any bonded tokens to be executed, but you can modify the script by changing `collateralAmount` to bond some tokens if you want. If you do so, remember to transfer the tokens to be bonded to the Relayer address, or the Autotask will fail.
+NOTE: You can at any time choose to _Bond to Keep3r Network_ again. This will allow you to bond additional KP3R tokens, but will reset the waiting period back to 3 days.
 
-After bonding, there is a three-day delay enforced by the network before you can start using your Keeper. You can check your Autotask runs logs to see how much time is remaining. Once that time has passed, the script will take care of *activating* your Relayer as a Keeper.
+## Activate your Relayer
+
+After the 3-day waiting period is over, you will have the option to _Activate as keep3r_. Doing so will send a transaction to activate your Relayer as a Keeper in the network.
+
+image::guide-keep3r-activated.png[Relayer is now a Keeper]
+
+## Create an Autotask
 
-NOTE: Once your Relayer is activated, there is no need for it to go through the same bonding and activation process again if you modify the Autotask that executes the jobs, or add another Autotask connected to it.
+Now that you have your Relayer active as a Keeper, let's set up the Defender Autotask that will instruct the Relay to send them. An Autotask is a snippet of code that runs at regular intervals, in this case, we'll use it to automate the process of bonding, activating, searching for jobs, and executing them.
```

Comment:
Should we refer to the app `Relay` or the users `Relayer`
```suggestion
Now that you have your Relayer active as a Keeper, let's set up the Defender Autotask that will instruct your Relayer to send them. An Autotask is a snippet of code that runs at regular intervals, in this case, we'll use it to automate the process of bonding, activating, searching for jobs, and executing them.
```

Reply:
+1 for this suggestion